docs/UserGuide.md

```diff
@@ -364,12 +393,19 @@ Parameter      | Description                   | Example
 ---------------|-------------------------------|--------------------------------
 `INDEX`        | The index number of the meeting you wish to edit in your displayed meeting list. <br><br>Detail:<br>It must be a positive integer. |`1`: Represents the 1st meeting in your list
 
+<div markdown="block" class="alert alert-warning">
+
+**:warning: Warning:** You cannot edit a meeting to have the same `TITLE`, `FROM` and `TO` as an existing meeting.
```

Comment:
I don't remember that we agreed to include this warning though. I don't think the user needs to know this as he/she won't go out of the way to try out this constraint. Should we delete all of these warnings to do with sameness (here and elsewhere)? I recall we didn't want any warnings in the commands (not even for delete, clear).

Reply:
yea thats what i remember also, im thinking whether we should delete all the warnings?